arangod/Agency/ResignLeadership.cpp

```diff
@@ -337,7 +344,28 @@ bool ResignLeadership::start(bool& aborts) {
 
       // Schedule shard relocations
       if (!scheduleMoveShards(pending)) {
-        finish("", "", false, "Could not schedule MoveShard.");
+        LOG_TOPIC("d4473", DEBUG, Logger::SUPERVISION)
```

Comment:
I would make this a "WARN" log level, according to our policy.

Reply:
Maybe "INFO", but it should show up in the logs whenever it happens in production, IMHO.

Reply:
I figured that it is enough to display this message when the job actually fails (see further down). Furthermore `scheduleMoveShards` already creates log messages which explain why it won't start yet.